content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -378,12 +384,33 @@ Run `GameLauncher.exe` from your project game release layout, which is located i
 
 - `C:\MyProject\install\bin\Windows\release\Monolithic` -- For monolithic projects.
 
-Now you can distribute your project to other Windows devices. To run the Game Launcher, you must install the [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160).
+
+## Distribute your build
+
+To distribute your build to other Windows computers:
+
+1. Create a zip folder of the `<install>\bin\Windows\release\<build>` folder. 
+
+    The contents of the zip folder should look similar to this: 
+
+        Monolithic
+        |   DevTestProject.GameLauncher.exe
+        |   PhysXDevice64.dll
+        |   PhysXGpu_64.dll
+        |   
+        \---Cache
+            \---pc
+            engine.pak
+                
+2. Distribute the zip folder in your preferred method such as through a file storage service. 
+
+3. To run the Game Launcher on the other Windows computers, it must have [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) installed.
```

Comment:
```suggestion
To run the Game Launcher on other Windows computers, they must have the [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) installed.
```
* I recommend dropping the ordered list numbering here, as this line is more of a note than an actionable step.
  * (Optional) I think that its appearance as a standard paragraph is fine, but this sentence might make the most sense [formatted as a note](https://www.o3de.org/docs/contributing/to-docs/style-guide/shortcodes/#note).
* Dropped unnecessary "the" ("other Windows computers").
* Presuming that "it" is meant to refer to "Windows computers," I changed it to "they." **Confirm if that's accurate.**
* Added "the" in front of "[MSVC" ("the [MSVC] Redistributable"), per phrasing in the linked MS docs.